CMakeLists.txt

```diff
@@ -1360,8 +1354,7 @@ _add_if("brotli"        HAVE_BROTLI)
 _add_if("zstd"          HAVE_ZSTD)
 _add_if("AsynchDNS"     USE_ARES OR USE_THREADS_POSIX OR USE_THREADS_WIN32)
 _add_if("IDN"           HAVE_LIBIDN2)
-_add_if("Largefile"     (CURL_SIZEOF_CURL_OFF_T GREATER 4) AND
-                        ((SIZEOF_OFF_T GREATER 4) OR USE_WIN32_LARGE_FILES))
+_add_if("Largefile"     ((SIZEOF_OFF_T GREATER 4) OR USE_WIN32_LARGE_FILES))
```

Comment:
Are you positive this is okay? Since `curl_off_t` is decoupled from `off_t`, they *can* have different sizes and that's why we checked both to begin with.